reproductions/driver-adapters/package.json

```diff
@@ -22,11 +22,11 @@
   "dependencies": {
     "@neondatabase/serverless": "0.6.0",
     "@planetscale/database": "1.11.0",
-    "@prisma/adapter-neon": "5.5.2",
-    "@prisma/adapter-pg": "5.5.2",
-    "@prisma/adapter-planetscale": "5.5.2",
+    "@prisma/adapter-neon": "5.6.0",
+    "@prisma/adapter-pg": "5.6.0",
+    "@prisma/adapter-planetscale": "5.6.0",
     "@prisma/client": "../../packages/client",
-    "@prisma/driver-adapter-utils": "5.5.2",
+    "@prisma/driver-adapter-utils": "5.6.0",
```

Comment:
Should these use the local packages via workspace now @Jolg42?

Reply:
It's not important here really, this code is unused in CI / not tested.
See https://github.com/prisma/prisma/pull/21914

Reply:
Why do we have them in our codebase then?

Reply:
I think the name should be changed, its purpose is not actually for doing reproductions as what we mean usually.

It's for
>See "Setting up a locally-linked development folder" in [CONTRIBUTING.md](../CONTRIBUTING.md)

:warning: This directory does not run in CI / GitHub Actions, it is meant only for local development.

Reply:
It will be renamed here https://github.com/prisma/prisma/pull/21914